### ISSUE TITLE
Parser reports 'bind:' name cannot be empty for bind:key

### ISSUE THREAD

**Issue by chbert** [pheuter/svelte-check-rs#42]
## Description

The parser incorrectly reports that `bind:` directive name is empty when using `bind:key` on a component.

## Minimal Reproduction

```svelte
<script lang="ts">
  import FieldCheckbox from './field-checkbox.svelte';
  
  let customKey = $state('default');
  let customValue = $state(false);
</script>

<FieldCheckbox
  bind:value={customValue}
  bind:key={customKey}
/>
```

Where `FieldCheckbox` has a `key` prop that can be bound:

```svelte
<!-- field-checkbox.svelte -->
<script lang="ts">
  let { value = $bindable(false), key = $bindable('') } = $props();
</script>

<input type="checkbox" bind:checked={value} />
<span>{key}</span>
```

## Error

```
Error: invalid directive: `bind:` name cannot be empty (parse-error)
```

## Expected Behavior

`bind:key` should be recognized as a valid two-way binding to a component's `key` prop, just like `bind:value`.

## Notes

`key` is a valid prop name in Svelte 5 components. While it has special meaning in `{#each}` blocks, it should be bindable as a component prop.

## Environment

- svelte-check-rs: 0.5.3
- Svelte: 5.45.2
